conf/messages

```diff
@@ -320,3 +320,13 @@ hasFurtherInformation.pageTitle = Do you want to tell us anything else about the
 hasFurtherInformation.heading = Do you want to tell us anything else about the underpayment?
 hasFurtherInformation.hint = That you have not told us about already.
 hasFurtherInformation.error.required = Select yes if you want to provide more information about the underpayment
+
+# Representative name
+representativeName.pageTitle = What is the importer’s name?
+representativeName.pageHeader = What is the importer’s name?
+representativeName.hint = We will send a copy of the demand for payment note to the importer by post. We will use this name on the envelope and letter.
+representativeName.error.nameNonEmpty = Enter the importers name
```

Comment:
This error does not match with the Content Spreadsheet. Please check with Nikki.